oak_functions/sdk/Cargo.lock

```diff
@@ -692,18 +639,23 @@ dependencies = [
 ]
 
 [[package]]
-name = "oak_attestation_common"
+name = "oak_abi"
```

Comment:
I am very skeptical of this dependency. Could you please double check that `oak_abi` is needed?